aio/tools/transforms/config.js

```diff
@@ -1,8 +1,8 @@
 const { resolve } = require('path');
 const { readdirSync } = require('fs');
 
-const BAZEL_OUTPUT_PATH = process.env.BAZEL_DGENI_OUTPUT_PATH;
-const PROJECT_ROOT = resolve(__dirname, '../../..');
+const BAZEL_OUTPUT_PATH = process.env.BAZEL_DGENI_OUTPUT_PATH || "dist";
```

Comment:
Nit: 
```suggestion
const BAZEL_OUTPUT_PATH = process.env.BAZEL_DGENI_OUTPUT_PATH || 'dist';
```

So, `dist` here is not `angular/aio/dist/`, right? It is somewhere inside the bazel sandbox?

Reply:
That's right. It actually ends up within in `angular/dist` within the runfiles in the sandbox, but that gets thrown away and is never written under the source tree.